src/measure_theory/measurable_space.lean

```diff
@@ -681,6 +691,84 @@ instance {α} {β : α → Type*} [m : Πa, measurable_space (β a)] : measurabl
 
 end constructions
 
+/-- A map `f : α → β` is called a *measurable embedding* if it is injective, measurable, and sends
+measurable sets to measurable sets. The latter assumption can be replaced with “`f` has measurable
+inverge `g : range f → α`”. -/
```

Comment:
```suggestion
inverse `g : range f → α`”. -/
```
Also, do you prove the equivalence between these two points of view somewhere? If so, it would be good to mention the name of this lemma here.